Update readmes to match common design (Map category) 2/4 (#466)
Esri/arcgis-runtime-samples-java: map/generate-offline-map-overrides/README.md

# Generate offline map (overrides)

Take a web map offline with additional options for each layer.

> ![Image of generate offline map overrides](GenerateOfflineMapOverrides.png)

Reviewer: Either describe what is actually depicted in the image or leave blank.

Reviewer 2: @tschie just to let you know that these alt texts have been approved by the accessibility team at Esri: the samples team went through a big effort of adding them all in a standardised way. We thought also that describing what was in the image was the way to go but turns out for Esri guidelines that's not needed for this.

## Use case

> When taking a web map offline, you may adjust the data (such as layers or tiles) that is downloaded by using custom parameter overrides. This can be used to reduce the extent of the map or the download size of the offline map. It can also be used to highlight specific data by removing irrelevant data. Additionally, this workflow allows you to take features offline that don't have a geometry - for example, features whose attributes have been populated in the office, but still need a site survey for their geometry.

Reviewer: ```suggestion
When taking a web map offline, you may adjust the data (such as layers or tiles) that are downloaded by using custom parameter overrides. This can be used to reduce the extent of the map or the download size of the offline map, or to highlight specific data by removing irrelevant data. Additionally, this workflow allows you to take features offline that don't have a geometry; for example, features whose attributes have been populated in the office, but still need a site survey for their geometry.
```

## How to use the sample

Sign in with an ArcGIS Online account when prompted for credentials (taking web maps offline requires an account) and modify the overrides parameters:

* Use the min/max scale input fields to adjust the level IDs to be taken offline for the streets basemap.
* Use the "Extent Buffer Distance" input field to set the buffer radius for the streets basemap.
* Check the checkboxes for the feature operational layers you want to include in the offline map.
* Use the "Min Hydrant Flow Rate" input field to only download features with a flow rate higher than this value.
* Select the "Water Pipes" checkbox if you want to crop the water pipe features to the extent of the map.

> After you have set up the overrides to your liking, click the "Generate offline map" button to start the download. A progress bar will display. Click the "Cancel" button if you want to stop the download. When the download is complete, the view will display the offline map. Pan around to see that it is cropped to the download area's extent.

Reviewer: ```suggestion
After you have set up the overrides to your liking, click the "Generate offline map" button to start the download. A progress bar will display. Click the "Cancel" button if you want to stop the download. When the download is complete, the view will display the offline map. Pan around to see that the map is cropped to the download area's extent.
```

Take a web map offline with additional options for each layer.

> ![](GenerateOfflineMapOverrides.png)

Reviewer 2: alt text needs to come back in (maybe this is already addressed with that other PR: same for other samples here)

Author: Yep, I've restored these here
